linux-tools/rng_tools/rng-tools.sh

```diff
@@ -94,6 +82,37 @@ function tc_local_cleanup()
 ################################################################################
 
 #
+#Check TPM device.
+#
+function check_rng()
+{
+
+        tc_register "checking TPM  device"
+        tc_exists /dev/random /dev/urandom /dev/tpm*
+        if [ $? -ne 0 ]; then
```

Comment:
Few doubts here:

Evenif /dev/tpm is not present , random number generator works with /dev/urandom or with /dev/hwrng.
I could verify in system which doesnt have /dev/tpm and successfully start rngd using "rngd -f"

Can you please verify again if /dev/tpm is must here.